free-programming-books.md

```diff
@@ -1916,6 +1916,7 @@ For resources on Angular, Backbone, D3, Dojo, Ember, Express, jQuery, Knockout,
 * [Think Complexity](http://greenteapress.com/complexity/) - Allen B. Downey (2nd Edition) (PDF, HTML)
 * [Web2py: Complete Reference Manual, 6th Edition (pre-release)](http://web2py.com/book) (2.5 - 2.x)
 * [Wikibooks: Python Programming](https://en.wikibooks.org/wiki/Python_Programming) (2.7)
+* [The Coder's Apprentice: Learning Programming with Python 3](http://www.spronck.net/pythonbook/) (3.x)
```

Comment:
Remember to put titles in alphabetical order. Also, books like this should include the author name, since there are many similar titles, and because the authors deserve credit. Thanks for working on the PR!